src/main/java/hudson/plugins/parameterizedtrigger/NodeAction.java

```diff
@@ -34,23 +34,29 @@
 import java.util.List;
 
 /**
- * {@link Action} that restricts the job to a particular node 
+ * {@link Action} that restricts the job to a particular node
  * when a project is scheduled
  * Will cause a unique build for each different node if a job is already queued.
- * 
+ *
  * @author Chris Johnson
  */
 public class NodeAction extends InvisibleAction implements LabelAssignmentAction, Queue.QueueAction, BuildBadgeAction {
 	private final Label nodeLabel;
```

Comment:
SpotBugs is disabled in the plugin, but it would be a good practice to have `CheckForNull` annotations